sklearn/ensemble/forest.py

```diff
@@ -1678,6 +1737,12 @@ class ExtraTreesRegressor(ForestRegressor):
         and add more estimators to the ensemble, otherwise, just fit a whole
         new forest. See :term:`the Glossary <warm_start>`.
 
+    max_samples : int or float (default=None)
```

Comment:
add comma and remove paranthesis

Reply:
The `(default=value)` style is used everywhere else in the module, so I'd prefer to leave it as is for consistency.

Reply:
We are going to change all of this pretty soon. This is one occurrence less to look at :)